### PR TITLE
Geshi path can be configurable by setting $config->setGeshiPath(). Fixes #199

### LINKED ISSUE #199: require_once(geshi.php) in svnlook.php
The following problem has been found in my PHP error log

[include/svnlook.php](https://github.com/websvnphp/websvn/blob/master/include/svnlook.php)

```
PHP Warning: require_once(geshi.php): failed to open stream: No such file or directory in .../websvn/include/svnlook.php on line 758
```

(I will try to find an instance in my access.log)

### PR DESCRIPTION
**Pull request by danielmarschall**
Fixed #199

### CODE REVIEW

**Comment by danielmarschall** [comment]
Can you change the commit message? "$config" is missing there.

**Review by michael-o — COMMENTED** [review]
Why not use `_setPath()`?

**Comment by danielmarschall** [comment]
Sorry, I don't understand

**Comment by danielmarschall** [comment]
Is this the correct use?

        var $geshiScript = 'geshi.php';

        function setGeshiPath($path) {
                $this->_setPath(this->geshiScript, $path, 'geshi.php');
        }

        function getGeshiScript() {
                return $this->geshiScript;
        }

**Comment by michael-o** [comment]
> Is this the correct use?
> 
> ```
>     var $geshiScript = 'geshi.php';
> 
>     function setGeshiPath($path) {
>             $this->_setPath($this->geshiScript, $path, 'geshi.php');
>     }
> 
>     function getGeshiScript() {
>             return $this->geshiScript;
>     }
> ```

Yes, this is my understanding. Please give it a try.

**Comment by danielmarschall** [comment]
Ok. I have tested my current PR, and it works for me

File: include/distconfig.php [hunk 1]
```diff
@@ -418,6 +419,7 @@
 // This will look for README.md file on the path and render it.
 // The name of "README.md" isn't configurable for now to simply follow GitHub's conventions.
 
+// $config->setParsedownPath('/usr/share/php/Parsedown/Parsedown.php'); // optional. Use if you have Parsedown installed without PEAR/Composer
```
Comment (michael-o) [inline include/distconfig.php:422]: This looks wrong, only the path, not the script path

**Review by michael-o — CHANGES_REQUESTED** [review]

**Comment by danielmarschall** [comment]
Thank you. You are right. I fixed it.

File: include/distconfig.php (hunk 1 above)
Comment (danielmarschall) [reply include/distconfig.php]: Fixed in [7c42b89](https://github.com/websvnphp/websvn/pull/202/commits/7c42b89242efc45a5d7ac506fd5acfc66e0b0f06)

**Review by michael-o — APPROVED** [review]
This looks logically sound to me.

**Pull request closed without merging**